app/src/main/res/xml/appfilter.xml

```diff
@@ -581,7 +581,7 @@
     <item component="ComponentInfo{org.floens.chan/org.floens.chan.ui.activity.BoardActivity}" drawable="clover" />
 
     <!-- Coinbase -->
-    <item component="ComponentInfo{com.coinbase.android/com.coinbase.android.signin.IntroActivity}" drawable="coinbase" />
+    <item component="ComponentInfo{com.coinbase.android/com.coinbase.android.MainActivity}" drawable="coinbase" />
```

Comment:
Generally we want to keep both so people using the older versions can keep their icon as well.